src/create_learner.Rmd

```diff
@@ -464,6 +464,34 @@ getFeatureImportanceLearner.classif.randomForestSRC = function(.learner, .model,
 ```
 
 
+## Integrating out-of-bag predictions in the [%mlr] framework
+
+Some learners like random forest use bagging. Bagging means that the learner consists of a ensemble of several base learners and each base learner is trained with a different subsample or bootstrap sample from all observations. For predicting an observation, 
+that was used in the training process, one can predict the observation with all base learners, where this observation was not used 
+for training, because it was not sampled. These predictions are called out-of-bag (OOB) predictions and are not prone to 
+overfitting, as the predictions where not made with learners that used the observation for training. 
```

Comment:
This whole text block (l. 469 - 472) is also on the Out-of-bag-predictions page.
I would remove it here and just concentrate on the technical stuff (starting with l. 474).